Sources/Sentry/SentryBreadcrumbStore.m

```diff
@@ -25,41 +25,53 @@
 @interface SentryBreadcrumbStore ()
 
 @property(nonatomic, strong) SentryFileManager *fileManager;
-
+@property(nonatomic, strong) NSMutableArray<SentryBreadcrumb *> *breadcrumbs;
 @end
 
 @implementation SentryBreadcrumbStore
 
-- (instancetype)initWithFileManager:(SentryFileManager *)fileManager {
-    self = [super init];
-    if (self) {
+- (instancetype)init {
+    if (self = [super init]) {
         self.maxBreadcrumbs = 50;
-        self.fileManager = fileManager;
+        self.breadcrumbs = [NSMutableArray new];
     }
     return self;
 }
 
+//- (instancetype)initWithFileManager:(SentryFileManager *)fileManager {
+//    self = [super init];
+//    if (self) {
+//        self.maxBreadcrumbs = 50;
+//        self.fileManager = fileManager;
+//    }
+//    return self;
+//}
+
```

Comment:
I think it's fine to remove to unused code in this whole file/everywhere.

Reply:
checked all files for completely out-commented old methods, and removed them.